Entities/AHuman.cpp

```diff
@@ -988,7 +970,9 @@ bool AHuman::EquipDeviceInGroup(string group, bool doEquip)
                 if (m_pFGArm->HoldsSomething())
                 {
                     m_pFGArm->GetHeldDevice()->Deactivate();
-                    m_Inventory.push_back(m_pFGArm->ReleaseHeldMO());
+                    MovableObject *previouslyHeldItem = m_pFGArm->ReleaseHeldMO();
+                    if (previouslyHeldItem->GetsHitByMOs()) { previouslyHeldItem->Draw(g_SceneMan.GetMOIDBitmap(), Vector(), g_DrawNoMOID, true); }
```

Comment:
What's happening here?

Reply:
See #217
I no longer remember the details, but there was an issue where devices in this case weren't being removed from the moid layer, and that could screw up collision handling or something. This forces the device to undraw itself from the moid layer, fixing the crash.

Reply:
Remember the details and put in a comment I guess.